flexget/plugins/output/notify_sns.py

```diff
@@ -0,0 +1,113 @@
+from __future__ import unicode_literals, division, absolute_import
+
+import json
+import logging
+
+from flexget import options, plugin
+from flexget.event import event
+
+log = logging.getLogger('output.sns')
+
+DEFAULT_TEMPLATE_VALUE = json.dumps({
+    'entry': {
+        'title': '{{title}}',
+        'url': '{{url}}',
+        'original_url': '{{original_url}}',
+        'series': '{{series_name}}',
+        'series_id': '{{series_id}}',
+    },
+    'task': '{{task}}',
+})
+
+class SNSNotification(object):
+
+    """
+    Emits SNS notifications of entries
+
+    Optionally writes the torrent itself to S3
+
+    Example configuration::
+
+      notify_sns:
+        [aws_access_key_id: <AWS ACCESS KEY ID>] (will be taken from AWS_ACCESS_KEY_ID environment if not provided)
+        [aws_secret_access_key: <AWS SECRET ACCESS KEY>] (will be taken from AWS_SECRET_ACCESS_KEY environment if not provided)
+        [profile_name: <AWS PROFILE NAME>] (If provided, use this profile name instead of the default.)
+        aws_region: <REGION>
+        sns_topic_arn: <SNS ARN>
+        [sns_notification_template: <TEMPLATE] (defaults to DEFAULT_TEMPLATE_VALUE)
+
+    """
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'sns_topic_arn': {'type': 'string'},
+            'sns_notification_template': {'type': 'string', 'default': DEFAULT_TEMPLATE_VALUE},
+            'aws_access_key_id': {'type': 'string'},
+            'aws_secret_access_key': {'type': 'string'},
+            'aws_region': {'type': 'string'},
+            'profile_name': {'type': 'string'},
+        },
+        'required': ['sns_topic_arn', 'aws_region'],
+        'additionalProperties': False,
+    }
+
+    # this has to run near the end of the plugin chain, because we
+    # should notify after all other outputs.
+    @plugin.priority(0)
+    def on_task_output(self, task, config):
+        # verify that we actually support Boto 3
+        try:
+            import boto3
+        except ImportError as e:
+            log.debug("Error importing boto3: %s", e)
+            raise plugin.DependencyError("sns", "boto3", "Boto3 module required. ImportError: %s" % e)
```

Comment:
I think maybe we should do this on task start, that way it fails when we do a `--test` as well (output phase is disabled during test.)